pkg/controller/snatglobalinfo.go

```diff
@@ -248,6 +248,10 @@ func (cont *AciController) handleSnatNodeInfo(nodeinfo *nodeinfo.NodeInfo) bool
 			}
 			cont.log.Debug("SnatPolicy Name: ", name)
 			if len(snatpolicy.SnatIp) != 0 {
+				nodeSNATEntryFound := cont.checkIfPolicyApplied(nodename, name, snatpolicy.ExpandedSnatIps)
+				if nodeSNATEntryFound {
```

Comment:
Would it make sense to log here and in L380?

Reply:
Added debug log